Add product/marketing collab guidance (#834)
sourcegraph/handbook: content/product/product_management/index.md

Launch tier levels (L1, L2, and L3) are also an important term to be aware of, and the definitions as well as process/source of truth are documented on the [marketing launch page](../../marketing/product-marketing/marketing_launch_tiers.md).

## Collaboration

Product Management tends to collaborate with a lot of different groups. Beyond our shared [work as a team value](../../company/values.md#work-as-a-team), there are some teams where collaboration is especially important or can be unclear so we have documented high level practices around how we work together.

### Product Management & Marketing

> Product Managemetn and Marketing must work close together to successfully create and launch products. Because of this, it's important to be clear on what the shared and unique responsibilities are within the working relationship.

Reviewer: ```suggestion
Product Management and Marketing must work closely together to successfully create and launch products. Because of this, it's important to be clear on what the shared and unique responsibilities are within the working relationship.
```

> TODO add link to source of this info

Author: @serinadawn do you have a link or something where we can credit the inspiration for this?

Author: @serinadawn I'll close this thread, but if you think of something feel free to add in a follow-up PR.

```suggestion
```

> | Product Marketing is DRI                                                                                 | Product Management is DRI                                                        |

Reviewer: @jyavorska - where does the roadmap fit in? I see PM as the owner of the product roadmap, but PMM is consulted/can influence the roadmap.

Author: That's a good add @abschu, I'll add a row.

| -------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------- |

> | Customer and market-facing                                                                               | Development and product-facing                                                   |

Author: ```suggestion
| Customer and market-facing                                                                               | Development and product-facing                                                   |
| Defining the roadmap | Providing important market context to the roadmap |
```

Author: Cc @abschu

| Receives new product info and product updates from product management teams                              | Delivers new product info and product updates to product marketing team          |
| Responsible for aligning product packaging and messaging with market demands                             | Responsible for aligning product requirements with market demands                |
| Defines key value props and messaging of new and updated products to sales, marketing, and support teams | Delivers technical aspects of new and updated products to product marketing team |

> | Closest allies are product management and sales                                                          | Closest allies are the development team and product marketing                    |

Reviewer: ```suggestion
| Closest allies are product management, marketing, and sales                                                          | Closest allies are the development team and product marketing                    |
```

Launch tier levels (L1, L2, and L3) are also an important term to be aware of, and the definitions as well as process/source of truth are documented on the [marketing launch page](../../marketing/product-marketing/marketing_launch_tiers.md).

## Collaboration

> Product Management tends to collaborate with a lot of different groups. Beyond our shared [work as a team value](../../company/values.md#work-as-a-team), there are some teams where collaboration is especially important or can be unclear so we have documented high level practices around how we work together.

Reviewer 2: ```suggestion
Product Management collaborates with a lot of different groups. Beyond our shared [work as a team value](../../company/values.md#work-as-a-team), there are some teams where collaboration is especially important and where roles and responsibilities can be unclear. To help clarify this, we have documented high level practices around how we work together.
```

### Product Management & Marketing

Product Management and Marketing must work closely together to successfully create and launch products. Because of this, it's important to be clear on what the shared and unique responsibilities are within the working relationship.

TODO add link to source of this info

| Product Marketing is DRI                                                                                 | Product Management is DRI                                                        |
| -------------------------------------------------------------------------------------------------------- | -------------------------------------------------------------------------------- |

> | Customer and market-facing                                                                               | Development and product-facing                                                   |

Reviewer 2: I worry that this doesn't fully represent the closeness of our Product Management team to our existing users. Would like to capture that a bit more.

Author: ```suggestion
| [Customer-first](../../company/values#customer-first) with orientation towards market                                                                               | [Customer-first](../../company/values#customer-first) with orientation towards product and engineering                                                  |
```

> | Customer and market-facing                                                                               | Development and product-facing                                                                         |

Author: ```suggestion
| [Customer-first](../../company/values#customer-first) with orientation towards market                                                                               | [Customer-first](../../company/values#customer-first) with orientation towards product and engineering    
```

Author: @christinaforney the other comment thread ended up in a weird state so remade it here. Do you feel like this captures the customer-facing element of both teams better?

Reviewer 2: Yes! I like that a lot more.

> | Defining the roadmap                                                                                     | Providing important market context to the roadmap                                |

Reviewer 2: I'm not sure that this one is accurate of the stat we are at today. I think Product is currently owning defining the roadmap.

Author: Oops, you're right! I swapped the two columns.

```suggestion
| Providing important market context to the roadmap                                                                                     | Defining the roadmap                                |
```

| Receives new product info and product updates from product management teams                              | Delivers new product info and product updates to product marketing team          |
| Responsible for aligning product packaging and messaging with market demands                             | Responsible for aligning product requirements with market demands                |
| Defines key value props and messaging of new and updated products to sales, marketing, and support teams | Delivers technical aspects of new and updated products to product marketing team |

> | Closest allies are product management, marketing, and sales                                              | Closest allies are the development team and product marketing                    |

Reviewer 2: ```suggestion
| Closest allies are product management, marketing, and sales                                              | Closest allies are the development team, customer support, customer engineering, and product marketing                    |
```
